src/pages/login/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import axios from "axios";
+import Router from "next/router";
+import { Form, Input, Button, Checkbox } from "antd";
+import { UserOutlined, LockOutlined } from "@ant-design/icons";
+import Link from "next/Link";
+
+import style from "./login.module.css";
+
+interface Value {
+  email: string;
+  password: string;
+}
+
+const Login = () => {
+  const auth = async (value: Value) => {
+    try {
+      await axios
+        .post("https://reactive.loca.lt/login/", {
+          email: value.email,
+          password: value.password,
+        })
+        .then(function (res) {
```

Comment:
you don't need `then` if you are using `async/await` syntax, just continue the logic as if a synchronous function is called